docs/tokenomics/token-holders/nns-app-quickstart.md

```diff
@@ -264,6 +264,54 @@ To disburse a neuron and return its ICP utility tokens:
 
 - #### Step 4:  Verify the transaction information, then click **Confirm and Send**. 
 
+## Neuron following
+
+To maximize neuron rewards and vote on every proposal submitted to the NNS, neuron holders can make use of the NNS's liquid democracy. It may be hard for neuron owners to manually vote on every single proposal submitted or neuron owners may not have the necessary level of technical understanding to evaluate proposals themselves. To aid in this, the NNS supports **neuron following**.
+
+A neuron can be configured to vote automatically by following the voting decision made by a group of other neurons. Following is done on the basis of topics, such that a neuron can choose to follow the voting decision for different proposal topics, such as governance or subnet management. For example, if the neuron has been configured to follow a group's decision for governance proposals, if the group votes to adopt a governance proposal with more than half of the followees voting in favor of adoption, the neuron following that group will vote to approve the proposal as well. If the neuron has been configured to follow a group's decision for subnet management proposals, if the group votes to reject a subnet management proposal with more than half of the followees voting to reject, the neuron following the group will vote to reject the proposal. 
+
+:::caution
+It is important to note that when following a group of neurons, your neuron will only cast a vote if at least 6 out of 10 of the neurons within the followed group vote the same way. If only 5 neurons vote, your neuron will not cast a vote. Therefore, it is recommended to follow groups with at least 10 followees.
```

Comment:
```suggestion
It is important to note that when following a group of neurons, your neuron will only cast a vote if more than half of the neurons within the group vote the same way. For example, if you follow 10 neurons on a topic, your neuron only votes if at least 6 out of the 10 neurons vote the same way. If only 5 neurons vote, your neuron will not cast a vote. This should be considered when choosing to follow multiple neurons on a topic.
```

I think the example is great, but 10 was just a random example. I would refrase it slightly to make it more general and then add one concrete example. I think the conclusion sentence in the end was not right (or I did maybe not fuly follow it) therefore I also proposed an alternative sentence there